main_app/igdb_api.py

```diff
@@ -71,14 +82,16 @@ def api_get_games_list(self, page: str = '1', **kwargs) -> list:
             'fields': 'name,cover,version_title,rating',
             'limit': self.__limit,
             'offset': (int(page) - 1) * self.__limit}
-        if kwargs.get('search') is None:
+        if not kwargs.get('search'):
             additional['order'] = 'popularity:desc'
         filters = self.__generate_filters(**kwargs)
         additional.update(filters)
+        print(additional)
         encoded_url = urlencode(additional, quote_via=quote_plus)
         data = self.__api_get(f'{category}{encoded_url}')
         images_query = list(set(map(lambda each: each.get('cover', ''), data)))
-        images_query.remove('')
+        if '' in images_query:
+            images_query.remove('')
```

Comment:
`images_query = list(set(each.get('cover') for each in data if each.get('cover')))`
Then there won't be `' '` inside list